fedimint-testing/src/federation.rs

```diff
@@ -109,7 +110,10 @@ impl FederationTest {
 
             let instances = config.consensus.iter_module_instances();
             let decoders = server_init.available_decoders(instances).unwrap();
-            let db = Database::new(MemDatabase::new(), decoders);
+            let db = Database::new(
+                fedimint_rocksdb::RocksDb::open(tempfile::tempdir().unwrap()).unwrap(),
+                decoders,
+            );
```

Comment:
can't repro with only client db changed (with server memdb)
and I thought this was client side deadlock, maybe it is server side?